ledger/metrics/src/main/scala/com/daml/metrics/DatabaseMetrics.scala

```diff
@@ -14,11 +15,59 @@ class DatabaseMetrics private[metrics] (
 ) extends MetricHandle.Factory {
   protected val dbPrefix: MetricName = prefix :+ name
 
+  @MetricDoc.Tag(
+    summary = "The time needed to acquire a connection to the database.",
+    description = """SQL statements are run in a dedicated executor. This metric measures the time
+                    |it takes between creating the SQL statement corresponding to the <operation>
+                    |and the point when it starts running on the dedicated executor.""",
+    qualification = Debug,
+  )
   val waitTimer: Timer = timer(dbPrefix :+ "wait")
+
+  @MetricDoc.Tag(
+    summary = "The time needed to run the SQL query and read the result.",
+    description = """This metric encompasses the time measured by `query` and `commit` metrics.
+                    |Additionally it includes the time needed to obtain the DB connection,
+                    |optionally roll it back and close the connection at the end.""",
+    qualification = Debug,
+  )
   val executionTimer: Timer = timer(dbPrefix :+ "exec")
+
+  @MetricDoc.Tag(
+    summary = "The time needed to turn serialized Daml-LF values into in-memory objects.",
+    description = """Some index database queries that target contracts and transactions involve a
+                    |Daml-LF translation step. For such queries this metric represents the time it
```

Comment:
```suggestion
                    |Daml-LF translation step. For such queries this metric stands for the time it
```
I mussed it on the sheet, but the word "represent" is repeated twice in one sentence, which sounds odd.